### ISSUE TITLE
Problem with show method for DenseEuclideanMetric

### ISSUE THREAD

**Issue by andreasnoack** [TuringLang/AdvancedHMC.jl#105]
```julia
julia> AdvancedHMC.DenseEuclideanMetric(Matrix{Float64}(I, 3, 3))
DiagEuclideanMetric([1.0, 1.0, 1.0])
```

**Comment by yebai** [comment]
Thanks for spotting this. This issue should be fixed by https://github.com/TuringLang/AdvancedHMC.jl/pull/106.

**Comment by yebai** [comment]
@xukai92 does the show method for metric always print the diagonal only?

**Comment by xukai92** [comment]
Yes it only print the diagonal. How do you think to warn the user?

I guess we could do something like `DenseEuclideanMetric(diag=[1.0, 1.0, 1.0])`?

**Comment by yebai** [comment]
> I guess we could do something like DenseEuclideanMetric(diag=[1.0, 1.0, 1.0])?

That looks good to me!

**Comment by xukai92** [comment]
Closed by #107
